### DISCUSSION TITLE
Exporting atlas slice on top of microscope image

### DISCUSSION THREAD

**Discussion by WL-So** [JingyiGF/HERBS#51] [General]
Stumbled across the program and was really drawn into this and playing around with HERBS has been a blast. As a whole, my lab was hoping to find an efficient way to highlight neuroanatomical regions on our brain slice images in a non-biased manner and HERBS condenses this process all into one specialised program. Though when the time came to export our images, we were not able to find a way to keep the brain regions we highlighted on our microscope images. It was possible to export either the atlas-generated slice or the transformed microscope image, but not both superimposed onto each other. Would it be possible to have enable such a function?

I've attached an image below to better illustrate what I mean, but without the triangulation points on the image too!
![HERBSimage_ArcuateN1](https://github.com/JingyiGF/HERBS/assets/152365300/3ee5dbab-678d-45b0-b58a-876a3cdadf22)

**Comment by JingyiGF** [comment]
Hi @WL-So ,

Sorry for the late reply, I have been sick for some days. 

I see what you mean. Actually the pyqtgraph already gave you this option. On the Scene from where you want to save the images, right-click on your mouse, you will see "Export". 
<img width="416" alt="Screenshot 2023-12-05 at 21 34 44" src="https://github.com/JingyiGF/HERBS/assets/64961126/bdac9b3c-4738-473c-a473-8946654ba843">

Best,
Jingyi

**Comment by WL-So** [comment]
HI @JingyiGF,

Thanks so much! Apologies for the late reply on my end too, I was overseas to spend the holidays. I'll give this a go! How were you able to hide the triangulation points on your image before exporting? The best I could do was colour them black.

**Comment by JingyiGF** [comment]
You may uncheck the triangulation button, or click any other tools. Then triangulation points will hide.On 10 Jan 2024, at 01:23, WL-So ***@***.***> wrote:﻿
HI @JingyiGF,
Thanks so much! Apologies for the late reply on my end too, I was overseas to spend the holidays. I'll give this a go! How were you able to hide the triangulation points on your image before exporting? The best I could do was colour them black.

**Reply by WL-So** [reply]
Thank you so much! Managed to get it working - thanks again for the support sincerely.
